bpf/init.sh

```diff
@@ -301,6 +303,38 @@ function bpf_load()
 	return $RETCODE
 }
 
+function bpf_load_cgroups()
+{
+	OPTS=$1
+	IN=$2
+	OUT=$3
+	PROG_TYPE=$4
+	WHERE=$5
+	SEC=$6
+	CALLS_MAP=$7
+	CGRP=$8
+
+	OPTS="${OPTS} ${OPTS_DIR} -DCALLS_MAP=${CALLS_MAP}"
+	bpf_compile $IN $OUT obj "$OPTS"
+
+	TMP=$(mktemp -d $RUNDIR/bpf.XXXXXXXXXX)
+	TMP_FILE="$TMP/x"
+	mount -t bpf bpf $TMP
```

Comment:
What's the motivation behind this obscuring of the pinned programs?

Reply:
> What's the motivation behind this obscuring of the pinned programs?

Not obscuring, I didn't want to make any assumption on where we have the regular bpf fs mount as all I'd need is just a temp location to pin it and the hook it up in the kernel.

Reply:
Don't we already mount a bpffs even if one isn't supplied? See `pkg/bpf/bpffs_linux.go:mountFs()`

Reply:
> Don't we already mount a bpffs even if one isn't supplied? See `pkg/bpf/bpffs_linux.go:mountFs()`

Ok, I'll see to pass it as an arg into init.sh.